### PR TITLE
Change test user

### PR DESCRIPTION
**Pull request by macburgee1**
Tests are failing because they're trying to use a non-existent user from directory, 'test-user2'. 

This issue also seeks to change the test user from 'test-user2' to 'utesting3', which exists in the directory test environment.

Also, per Brett B:

> Brett B 2:43 PM (4/6/2020)
> We do not provision test accounts to our production identity environment, but we DO have a test environment that can be used for testing.

Affected tests:
- Drupal\unl_user\Tests\ImportUserTest::testImportUser()
- Drupal\unl_user\Tests\PersonDataQueryTest::testGetUserData()

~~This issue seeks to add a production/test mode. The toggle would affect which server is queried (directory.unl.edu or directory-test.unl.edu).~~

### CODE REVIEW

**Comment by macburgee1** [comment]
This PR is blocked until #1 is merged.

Remaining tasks:
1. Update tests to use 'utesting3' user
1. Update tests to set module to test mode before running
1. `Drupal\unl_user\Tests::PersonDataQueryTest()` will need to be a kernel test instead of a unit test

**Comment by ericras** [comment]
Looks like utesting3 exists in the production environment too. And it has more attributes (like major) set in the production environment than the testing one.

Actually, up until just now when I changed it, directory-test was pointed at the production identity environment. Since we just have read-only access, it makes more sense to look at production data when we're testing our directory changes (as opposed to testing IAM's updates to the service).

**Review by ericras — APPROVED** [review]

**Comment by ericras** [comment]
Didn't mean to hit approve since there's more changes you were going to make.

**Comment by ericras** [comment]
hhusker1 is now available as a test user: directory.unledu/people/hhusker1

**Comment by macburgee1** [comment]
Since we have now have a test user on production, there's no value in being to toggle between production/test mode. I rebased the issue branch and removed the prior commit. This branch now simply replaces the test user with 'hhusker1'.

There's a school thought that tests shouldn't rely on external services and should mock any needed requests. In general, this is sound guidance; however, since we own both this module and the external service, directory.unl.edu, I think we're safe to test again directory.unl.edu.

@ericras - Thanks for adding hhusker1.

Test results from local:
```
$ ../vendor/phpunit/phpunit/phpunit -c core modules/contrib/unl_user/ --verbose
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.2.28-4+0~20200224.38+debian10~1.gbp1ca010
Configuration: /var/www/html/web/core/phpunit.xml.dist

Testing modules/contrib/unl_user/
..                                                                  2 / 2 (100%)

Time: 1.26 minutes, Memory: 4.00 MB

OK (2 tests, 80 assertions)

```

**Review by ericras — APPROVED** [review]
Yes, I think you're right about not needing directory-test because if anything mocking the request would be better anyway.

**Pull request merged**
